### PR TITLE
Add recipe for ipysankeywidget

### PR DESCRIPTION
**Pull request by ricklupton**

### CODE REVIEW

**Comment by ricklupton** [comment]
Ping @conda-forge/staged-recipes - this has been here a while, could you take a look please?

**Comment by ocefpaf** [comment]
> Ping @conda-forge/staged-recipes - this has been here a while, could you take a look please?

Please see https://github.com/conda-forge/staged-recipes#12-my-pull-request-passes-all-checks-but-hasnt-received-any-attention-how-do-i-call-attention-to-my-pr--what-is-the-customary-amount-of-time-to-wait

File: recipes/ipysankeywidget/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    - ipywidgets >=7.0.0
+
+test:
+  imports:
+    - ipysankeywidget
+
+about:
+  home: http://github.com/ricklupton/ipysankeywidget
+  license: MIT
+  license_family: MIT
```
Comment (ocefpaf) [inline recipes/ipysankeywidget/meta.yaml]: Is the license file in the source tarball? If so can you add it here?
If not can you add it manually and open an issue upstream?

File: recipes/ipysankeywidget/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1f3a1ab7481ad41c06f05e7524479109d6f5685371a85b800667253221c25fa4
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```
Comment (beckermr) [inline recipes/ipysankeywidget/meta.yaml]: You should use the build strong from the example recipe.

**Comment by beckermr** [comment]
We have to ship the license file too.

File: recipes/ipysankeywidget/meta.yaml (hunk 2 above)
Comment (ricklupton) [reply recipes/ipysankeywidget/meta.yaml]: Thanks, do you mean [this example](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L33): `{{ PYTHON }} -m pip install . -vv`?

I thought I was consistent with [the FAQ](https://github.com/conda-forge/staged-recipes#7-when-or-why-do-i-need-to-use--python---m-pip-install---no-deps---vv): `{{ PYTHON }} -m pip install --no-deps . -vv`

**Comment by ricklupton** [comment]
> Please see https://github.com/conda-forge/staged-recipes#12-my-pull-request-passes-all-checks-but-hasnt-received-any-attention-how-do-i-call-attention-to-my-pr--what-is-the-customary-amount-of-time-to-wait

@ocefpaf Thanks for the comments and sorry if it sounded like I was complaining this hadn't had attention, it's been on my list to come back to for a while!

**Comment by ricklupton** [comment]
> We have to ship the license file too.

Did you mean I should add the license file to the PR?

**Comment by ocefpaf** [comment]
> Did you mean I should add the license file to the PR?

Yep. And please open an issue upstream. PyPI does not enforce the license to be present and that is a bad practice IMO, they get away with it b/c the publishers are 99.99% of the time the authors themselves. In conda-forge the case is the opposite, most of the time the one publishing the artifact is a member of the community, so the license is mandatory to avoid when the license text requests it to be present. (Which is the case here.)

**Comment by ricklupton** [comment]
Thanks @ocefpaf @beckermr for the feedback and quick response. I'll add the license file to my packages upstream too.

I'm about to prepare a recipe for another package -- should I have done something differently with the build string?

**Comment by ocefpaf** [comment]
> Edit: I think I see now, as of 8 days ago the --no-deps workaround is no longer needed.

Yep. It does not hurt if that is there but you do not need to add it.

**Pull request merged**